stacks/apps/apple-api-bridge.yml

```diff
@@ -0,0 +1,349 @@
+# stacks/apps/apple-api-bridge.yml
+# 100A
```

Comment:
It doesn't look like this template has any hard dependencies on any other stacks, so it could go in the 100A template. But, if it had any soft/implicit dependencies, it could make sense to go in a higher-order template. Feeder is in 200A, so if it only ever really makes sense for the API Bridge to exist after Feeder, I'd probably put this in 300A. If it doesn't depend on Feeder (or anything else) in practice, it can stay in 100A.

Also worth noting that I don't see an addition to the 100A template in this PR. It will have to get added to one of those templates to actually get deployed.